components/dashboard/src/login/SSOLoginForm.tsx

```diff
@@ -4,41 +4,27 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { FC, useCallback, useEffect, useState } from "react";
+import { FC, useCallback, useState } from "react";
 import Alert from "../components/Alert";
 import { Button } from "../components/Button";
 import { TextInputField } from "../components/forms/TextInputField";
 import { useOnBlurError } from "../hooks/use-onblur-error";
 import { openOIDCStartWindow } from "../provider-utils";
+import { useFeatureFlag } from "../data/featureflag-query";
 
 type Props = {
+    singleOrgMode?: boolean;
     onSuccess: () => void;
 };
-export const SSOLoginForm: FC<Props> = ({ onSuccess }) => {
+export const SSOLoginForm: FC<Props> = ({ singleOrgMode, onSuccess }) => {
```

Comment:
I always saw this as a "pre-selected org" mode (condition: "when slug is present"). There are two sources for that slug:
 - the list of previously used orgs on this browser has length === 1
 - the "default org"

With that, we'd also solve the gitpod.io usecase. Especially, if we extend it:
 - with a way to go back from the "pre-selected" state to the "show input field" state
 - offer a nice drop-down / auto-completion / Google-style selector for the cases where length > 1

But also, we don't have to do all of this now, in this PR. :+1:

Reply:
Given the single-org situation is going to be the 90% case (maybe even 100%) for dedicated, I believe it is worth having just a button without a slug or drop down for this. Also, we don't have the APIs to ask for the single org slug in order to prefill it. 
We can do the remember the last used slug in addition to this change as it would rely on browser state and is therefore quite simple to do. But for dedicated it is not important, so I propose we do that later?

Reply:
>  But for dedicated it is not important, so I propose we do that later?

Absolutely, let's do that. :100:

Reply:
Cross-posting UX-13 for visibility. Cc @geropl @svenefftinge